modules/nw-udn-additional-config-details.adoc

```diff
@@ -37,8 +40,22 @@ When enabled, OVN-Kubernetes applies the IP configuration to the SDN infrastruct
 **Disabled:** +
 When disabled, OVN-Kubernetes only assigns MAC addresses and provides layer 2 communication, which allows users to configure IP addresses. `Disabled` is only available for layer 2 (secondary) networks. By disabling IPAM, features that rely on selecting pods by IP, for example, network policy, services, and so on, no longer function. Additionally, IP port security is also disabled for interfaces attached to this network. The `subnets` field must be empty when `spec.ipam.mode` is set to `Disabled.`
 
-|`spec.layer2.mtu` and `spec.layer3.mtu`
+|`spec.network.<topology>.mtu`
+|`spec.<topology>.mtu`
 |integer
 |The maximum transmission units (MTU). The default value is `1400`. The boundary for IPv4 is `576`, and for IPv6 it is `1280`.
 
-|====
\ No newline at end of file
+|`spec.network.localnet.vlan`
```

Comment:
Can we add entry for physicalNetworkName?

Reply:
Done